### DISCUSSION TITLE
Deleting the last workflow run re-enables manually disabled workflows

### DISCUSSION THREAD

**Discussion by nmschulte** [community/community#38653] [Actions]
Deleting the last workflow run for a workflow removes the [manually] disabled status, effectively re-enabling the workflow.

**Comment by ossanna16** [comment]
Hi there @nmschulte and welcome to our community! Can you please add a bit more detail regarding your issue so we can help?

**Reply by nmschulte** [reply]
Hi @ossanna16, and thanks.

A repository I work with and have cloned, has [quite a few] GitHub Actions workflows.  I want to disable [some of] these workflows in my clone.
I found I can do this by choosing to "Disable Workflow" from the overflow menu when viewing a specific workflow:
![image](https://user-images.githubusercontent.com/8540239/201213454-52e230fb-dfcf-4614-bb13-2036a70ede9b.png)

However, ...
1) workflows are only visible in the Actions tab if they have runs; they do not show if there are no runs, and ...
2) deleting the last run from a workflow seems to cause it to "forget" the "Manually disabled" state, such that pushing a new branch/commit (or presumably "triggering" the workflow) causes it to be re-enabled and run

A work-around is to simply leave one or more runs in the given workflow, though this may not last forever if e.g. historical workflows are automatically removed after some time.

**Reply by ossanna16** [reply]
Thank you for the additional context!
